Doc/reference/datamodel.rst

```diff
@@ -21,8 +21,8 @@ conformance to Von Neumann's model of a "stored program computer", code is also
 represented by objects.)
 
 .. index::
-   builtin: id
-   builtin: type
+   pair: built-in function; id
+   pair: built-in function; type
```

Comment:
This is a straight find/replace for what Sphinx does natively (uses the value 'built-in function'), but clearly some of these aren't functions (e.g. ``type`` here). I'd welcome thoughts on the best way to address this, if we should keep as is or try and use different verbiage.

A

Reply:
Even if technically it might not be a built-in function, if people are going to search for it as a built-in function and/or recognize it as the right `type` when they see "built-in function", then I think it's ok to use it even if technically it's inaccurate.